File: Java/프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[num_list.length]; //num_list만큼 방을 만들어준다
        int length = num_list.length;
        for(int i = 0; i<length; i++){
            //num_list 길이 - i - 1을 하면 반대로 숫자를 넣을 수 있다
            answer[length - i - 1] += num_list[i];
        }
        return answer;
    }
}

File: Python3/프로그래머스/0/120821. 배열 뒤집기/배열 뒤집기.py
def solution(num_list):
    #for문과 len을 이용해 answer에 방을 만들어두고
    answer = [0 for i in range(len(num_list))]
    length = len(num_list)
    #거꾸로 num_list의 숫자들을 넣는다
    for i in range(length):
        answer[length - i - 1] += num_list[i]
    return answer